hedera-node/src/test/java/com/hedera/services/contracts/execution/HederaTracerTest.java

```diff
@@ -100,6 +101,7 @@ void initializesActionsAsExpectedOnNewFrames() {
         assertNull(topLevelAction.getRecipientAccount());
         assertEquals(value.toLong(), topLevelAction.getValue());
         assertEquals(0, topLevelAction.getCallDepth());
+        assertEquals(OP_CALL, topLevelAction.getCallOperationType());
```

Comment:
Can you update `finalizesCodeSuccessfulCreateMessageFrameAsExpected()` to check for `assertEquals(OP_CREATE, childFrame1.getCallOperationType());`